x/leveragelp/types/codec.go

```diff
@@ -40,6 +41,9 @@ func RegisterInterfaces(registry cdctypes.InterfaceRegistry) {
 	registry.RegisterImplementations((*sdk.Msg)(nil),
 		&MsgAddCollateral{},
 	)
+	registry.RegisterImplementations((*sdk.Msg)(nil),
+		&MsgClosePositions{},
```

Comment:
Can put this line as well as line 42 below lines 38.

Reply:
why ?

Reply:
okay, you mean to remove additional function call